scripts/e2e_test_keras.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [[ $# -ne 6 ]] ; then
+  echo "Usage: $0 <path/to/keras/model> <path/to/emitc/include/files> <path/to/emitc-opt> <compiler> <seed> <output_dir>"
+  echo
+  echo "This script expects a python version in the PATH with a recent version of tensorflow installed."
+  echo "Tested with python 3.6.9 and tensorflow 2.4.0" 
+
+  exit 1
+fi
+
+KERAS_MODEL=$1
+EMITC_INCLUDE_DIR=$2
+EMITC_OPT=$3
+EMITC_TRANSLATE=$(dirname $EMITC_OPT)/emitc-translate
+CPP_COMPILER=$4
+SEED=$5
+BATCH_SIZE=2
+OUTPUT_DIR=$6
+
+echo "KERAS_MODEL=$KERAS_MODEL"
+echo "EMITC_INCLUDE_DIR=$EMITC_INCLUDE_DIR"
+echo "EMITC_OPT=$EMITC_OPT"
+echo "EMITC_TRANSLATE=$EMITC_TRANSLATE"
+echo "CPP_COMPILER=$CPP_COMPILER"
+echo "SEED=$SEED"
+echo "BATCH_SIZE=$BATCH_SIZE"
+echo "OUTPUT_DIR=$OUTPUT_DIR"
+
+echo "Setting up output directory"
+mkdir -p "$OUTPUT_DIR"
+
+echo "Copying over keras model"
+cp "$KERAS_MODEL" "$OUTPUT_DIR"/model.h5
```

Comment:
I don't see why copying is strictly necessary. At least it is used inconsistently.

Reply:
This is a left over from testing. I'll delete it.